website/docs/r/netlify_deploy_key.html.markdown

```diff
@@ -0,0 +1,34 @@
+---
+layout: "netlify"
+page_title: "Netlify: netlify_deploy_key"
+sidebar_current: "docs-netlify-resource-deploy-key"
+description: |-
+  Provides an deploy key resource.
+---
+
+# netlify_deploy_key
+
+Creates a new netlify deploy key, typically used by the `netlify_site` resource.
+
+## Example Usage
+
+```hcl
+resource "netlify_deploy_key" "key" {}
+
+resource "netlify_site" "main" {
+  // ...
+  repo {
+    // ...
+    deploy_key_id = "${netlify_deploy_key.key.id}"
+  }
+}
+```
+
```

Comment:
I'm not sure if it matters but there are quite a few new lines here that could render incorrectly